plugins/woocommerce/src/Blocks/BlockTypes/ProductStockIndicator.php

```diff
@@ -38,6 +38,29 @@ protected function get_block_type_uses_context() {
 		return [ 'query', 'queryId', 'postId' ];
 	}
 
+		/**
+	 * Get the product types that are allowed in the editor.
+	 *
+	 * @return array
+	 */
+	protected function product_types_without_stock_indicator() {
+		return apply_filters( 'woocommerce_blocks_product_types_without_stock_indicator', array( 'external', 'grouped', 'variable' ) );
```

Comment:
I wonder if we should mark this filter as experimental, at least for now. Or maybe even remove it until we find a use case where it's needed. (I'm sure there are, but I'm a bit wary of adding the filter so early in the development cycle when we hadn't started looking into extensibility yet). What do you think?